### DISCUSSION TITLE
Could someone give me an example to make an Embed Message?

### DISCUSSION THREAD

**Discussion by ambitaah** [Rapptz/discord.py#6523] [General] [answered]
Hello! Well, I'm new to python and I don't know the truth much, so I'm trying to make a discord bot with python, and I can't find any example of the Embed Message that works for me, since it throws an error or it doesn't work ...
**So if you can help me it would be appreciated !**

**Comment by AbstractUmbra** [comment, marked as answer]
You can construct an Embed by setting the attributes listed on the [docs](https://discordpy.readthedocs.io/en/latest/api.html?highlight=embed#discord.Embed) for them.

A brief example is here:

```py
embed = discord.Embed(url="https://www.google.com/", title="Google", colour=discord.Colour.default())
embed.add_field(name="Field 1", value="Value 1")
embed.add_field(name="Field 2", value="Value 2")
embed.timestamp = datetime.datetime.utcnow()
embed.set_footer(text="This is a footer.")

await channel.send(embed=embed)
```
The example above looks like [this](https://umbra-is.gay/303lhTopuA79GQ.png).
This is but a small example of an embed. There are more things you can set and use, so I recommend experimenting with them.

**Reply by ambitaah** [reply]
Thank you very much for the help !
